pkg/controller/embercsi/common.go

```diff
@@ -85,6 +85,10 @@ func generateEnvVars(ecsi *embercsiv1alpha1.EmberStorageBackend, driverMode stri
 	X_CSI_EMBER_CONFIG, err := interfaceToString(ecsi.Spec.Config.EnvVars.X_CSI_EMBER_CONFIG)
 	if err == nil {
 		setJsonKeyIfEmpty(&X_CSI_EMBER_CONFIG, "plugin_name", GetPluginDomainName(ecsi.Name))
+		setJsonKeyIfEmpty(&X_CSI_EMBER_CONFIG, "project_id", "ember-csi.io")
+		setJsonKeyIfEmpty(&X_CSI_EMBER_CONFIG, "user_id", "ember-csi.io")
+		setJsonKeyIfEmpty(&X_CSI_EMBER_CONFIG, "root_helper", "sudo")
+		setJsonKeyIfEmpty(&X_CSI_EMBER_CONFIG, "request_multipath", "true")
```

Comment:
I don't think this is all there is to it...

The option in the form is [`multipath`](https://github.com/embercsi/ember-csi-operator/blob/2d1e1de3de5aa1ed06f76eefb8b6fecb5e9be1fe/deploy/olm-catalog/0.9.4/ember-csi-operator.v0.9.4.clusterserviceversion.yaml#L8792-L8798), which is the right name for the controller container, and for controller and nodes it needs to be copied to `request_multipath` option.

Also the default is [`false`](https://github.com/embercsi/ember-csi/blob/b2ccacaf5675c8f6d7710109e7df877e2a371ada/ember_csi/defaults.py#L25).